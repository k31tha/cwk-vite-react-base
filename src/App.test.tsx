import {describe, it, expect} from 'vitest';

describe('basic tests', () => {
  it('true to be true', () => {
    expect(true).toBe(true);
  });
  it('toUpperCase', () => {
    const result = 'foobar'.toUpperCase();
    expect(result).toMatchSnapshot();
  });
  it('toLowerCase - inline', () => {
    const result = 'foobar'.toLowerCase();
    expect(result).toMatchInlineSnapshot('"foobar"');
  });
});
