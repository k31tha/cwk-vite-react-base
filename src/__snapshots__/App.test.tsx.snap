// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`basic tests > toUpperCase 1`] = `"FOOBAR"`;
